cluster-autoscaler/core/static_autoscaler.go

```diff
@@ -719,23 +719,23 @@ func (a *StaticAutoscaler) obtainNodeLists(cp cloudprovider.CloudProvider) ([]*a
 	return allNodes, readyNodes, nil
 }
 
-// actOnEmptyCluster returns true if the cluster was empty and thus acted upon
-func (a *StaticAutoscaler) actOnEmptyCluster(allNodes, readyNodes []*apiv1.Node, currentTime time.Time) bool {
```

Comment:
The behavior is still confusing.  Now `shouldActOnEmptyCluster` returns true if the cluster is not empty.